include/application/pokegear/configure/pokegear_configure_internal.h

```diff
@@ -0,0 +1,57 @@
+#ifndef GUARD_POKEHEARTGOLD_APPLICATION_POKEGEAR_CONFIGURE_POKEGEAR_CONFIGURE_INTERNAL_H
```

Comment:
can we have a more sane name pls (and one that follows the convention of the other ones)

Reply:
the name is a representation of the path relative to the includedir, as has been standard for this project

Reply:
sure, it's just this one does get kinda long but I guess that's fair, though I would prefer it to not have GUARD in front as the others don't, but it's not siginificant